cavern/src/intTest/java/org/opencadc/cavern/MetadataIntTest.java

```diff
@@ -118,6 +132,62 @@ public static void staticInit() throws Exception
             throw new IllegalStateException("expected system property " + uriProp + " = <base vos URI>, found: " + uri);
     }
 
+    @Test
+    public void testHeaderMetadata() throws Exception {
```

Comment:
Can you rename this test method (and maybe testContentMD5) so it is clear in a report of pass/fail (eg like we did for c3tp) that this tests the file download header and the other tests the node metadata header?

Maybe testFileDownloadContentMD5() and testNodeContentMD5()??

Reply:
Done